tl-tl/sc/systemfees.md

```diff
@@ -1,20 +1,20 @@
-# System Fees
+# Mga Bayad sa Sistema
 
-## Transaction fees
+## Mga bayad sa transaksyon
 
-There are currently no transaction fees. The user can however choose to pay transaction fee for priority.
+Kasalukuyang walang bayad ang mga transaksyon. Ang gumagmit, gayunpaman, ay maaring pumiling bumayad para sa prayoridad.
```

Comment:
- `gumagamit`
- you could remove commas `gayunpaman`

Reply:
fix provided